lms/migrations/versions/1693f6ade03d_enable_ms_one_drive.py

```diff
@@ -0,0 +1,28 @@
+"""
+Enable MS OneDrive on instances that have previously been disabled
+
+Revision ID: 1693f6ade03d
+Revises: 2fd66b9ab1a1
+Create Date: 2021-11-18 10:47:24.773542
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "1693f6ade03d"
+down_revision = "2fd66b9ab1a1"
+
+
+def upgrade():
+    conn = op.get_bind()
+
+    conn.execute(
+        """UPDATE application_instances SET settings = settings || jsonb '{"microsoft_onedrive":{"files_enabled": true}}'
+            WHERE settings -> 'microsoft_onedrive' -> 'files_enabled' = 'false';"""
```

Comment:
with this where we are only updating the ones that have no the setting to false.

Reply:
It's sometimes a good idea to add some logging to data migrations, e.g. to log how many rows were changed. It gives a way to see, when running the migration on production, that it did what we expected it to do. I'm not sure how you could add that logging when using a single `conn.execute()` like this though